HelloWorld.java

```diff
@@ -4,6 +4,7 @@ public static void main(String[] args) {
         // Prints "Hello, World" to the terminal window.
         System.out.println("Hello, World111");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:5660/coding_rules#rule_key=squid%3AS106)